incubator/hnc/internal/validators/namespace.go

```diff
@@ -79,7 +79,26 @@ func (v *Namespace) checkForest(req *nsRequest) admission.Response {
 
 	ns := v.Forest.Get(req.ns.Name)
 
-	// Early exit to allow non-delete requests or if the namespace allows cascading deletion.
+	// Allow all create namespace requests except if the namespace name already exists in
+	// the external hierarchy (the default webhook already denies existing namespace names).
```

Comment:
What's "the default webhook"?

Reply:
Updated to apiserver instead of default webhook. ( I thought the request is denied by default webhook but it turned out to be apiserver itself).